config.js

```diff
@@ -581,6 +581,7 @@ var config = {
      hiddenDomain
      ignoreStartMuted
      nick
+     opusMaxAvgBitrate
```

Comment:
Can you please move this to the [`Audio` section](https://github.com/jitsi/jitsi-meet/blob/master/config.js#L78) and add a description of what it does?

Reply:
Done. Please let me know if the comment should be adjusted/rephrased.